cdap-ui/app/directives/complex-schema/union-schema/union-schema.html

```diff
@@ -31,7 +31,7 @@
               <select class="form-control"
                       ng-model="item.displayType"
                       ng-options="option as option for option in UnionSchema.SCHEMA_TYPES"
-                      ng-change="UnionSchema.changeType(item)">
+                      ng-change="UnionSchema.changeType(item, '{{item.displayType}}')">
```

Comment:
the second argument can't just be `item.displayType`? does it have to be wrapped in the mustache syntax and quote?

Reply:
Yeah this is needed to get the `displayType` prop of the `item` before the change, instead of the one after